main.go

```diff
@@ -20,23 +20,27 @@ package main
 import (
 	"context"
 	"fmt"
-	"github.com/jessevdk/go-flags"
 	"os"
+	"strings"
 	"time"
+
+	"github.com/jessevdk/go-flags"
 )
 
 type options struct {
 	ProjectId  string `short:"p" long:"project" description:"(required) GCP Project ID."`
 	InstanceId string `short:"i" long:"instance" description:"(required) Cloud Spanner Instance ID."`
 	DatabaseId string `short:"d" long:"database" description:"(required) Cloud Spanner Database ID."`
+	Tables     string `short:"t" long:"tables" description:"comma-separated table names, e.g. \"table1,table2\" "`
```

Comment:
Can we remove short name `"t"` from the option? I want to reserve the short name for future use.